Add ETH address support into the AddressInput component

mraveux opened the pull request:
This PR add support for ETH address into the AddressInput components, and allow one to restrict the component to only accept ETH or NIM addresses.
(It also update the transifex config to support the new client)

sisou commented on package.json:
```diff
@@ -24,11 +25,13 @@
     "@nimiq/style": "^0.8.2",
     "@nimiq/utils": "^0.8.2",
     "big-integer": "^1.6.44",
+    "ethers": "^5.7.2",
```
That's a huge dependency for just checking address validity. There's smaller packages for that, such as
- https://www.npmjs.com/package/ethereum-address (really only that one function)
- https://www.npmjs.com/package/@ethersproject/address (part of ethers, with more stuff)

I would much prefer to use such a small package and include it in the bundle, and not require the using-project to install and bundle ethers downstream.

mraveux replied:
I guess it makes sense to use something smaller than `ethers` and only for address validation, but since the package is already loaded anyway in the wallet we though it would be alright to use it there. But now that I think about it, if someone else use the component outside of the wallet and use another library than `ethers` then it would become quite a heavy component just for address validation.
So I will change the component to actually include the function `isAddress` and `isChecksumAddress` since they are quite short, and now only use the `crypto-js` package who's still needed for the `sha3` method used in the `isChecksumAddress` function.

sisou reviewed (changes requested):
Nice work! I just would not include `ethers` as a full dependency, that makes no sense IMO.

mraveux commented:
Removed `ethers` dependency and now implemented `isEthAddress` and `isEthChecksumAddress` directly into the component and only importing the crypto-js dependency for the `sha3` function.
Let me know @sisou if that's good for you 😉

sisou commented:
Please also remove the `patch-package` and `postinstall` dependencies as they are now unused.

sisou commented on package.json:
```diff
@@ -75,5 +78,6 @@
   "license": "Apache-2.0",
-  "types": "types/index.d.ts"
+  "types": "types/index.d.ts",
+  "packageManager": "yarn@1.22.19"
```
Where does this line come fro or is required for?

mraveux replied:
the `packageManager` entry? It's used from nodejs 14 to enforce a package manager version to be used in the project https://nodejs.org/docs/latest-v14.x/api/corepack.html#corepack_configuring_a_package

Since using a newer yarn version to install the dependencies break things we decided with @danimoh to lock it to yarn 1 until we fixed the issues that comes with newer yarn version.

sisou replied:
It seems it's fixed to a specific version now, not yarn 1 in generell. Is that an issue if I don't have _exactly_ that version?

mraveux replied:
It should not be an issue if you have a lower version, and `1.22.19` is the last yarn 1 version apparently.
Unfortunately I don't think we can provide a more "general" version like `yarn@1.x.x` or `yarn@^1.22.19` since it doesn't seems to be supported yet.

This entry should anyway only be used by node if you have `corepack` enabled, like I do, and installed your package manager using it. And it will tell nodejs to automatically use the correct yarn version in this repo, even if you have yarn 3 globally installed.

sisou replied:
Ah, thanks! 👍
